src/styles.scss

```diff
@@ -88,4 +88,27 @@ label {
   color: $yonder-pink;
 	border: 1px solid $yonder-pink;
 	margin-left: 20px;
+}
+
+/* width */
+::-webkit-scrollbar {
+    width: 12px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: $yonder-light-gray; 
+  border-radius: 6px;
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+    border-radius: 6px;
+    background: $yonder-pink; 
+}
+
+/* Handle hover */
+::-webkit-scrollbar-thumb:hover {
```

Comment:
It looks like you have the same style for` ::-webkit-scrollbar-thumb` and `::-webkit-scrollbar-thumb:hover `